apps/salga-mobile-api/README.md

```diff
@@ -0,0 +1,22 @@
+### Steps to deploy Salga Mobile API
+____________________________________
+
+The following steps assume that you have familiarized yourself with instructions of the README located at the
+root of the [OpenUpSA/ansible-config](https://github.com/OpenUpSA/ansible-config) repo.
+
+For commands below, ensure you make use of appropriate inventory flag depending on environment and substitute out
+`prod` for correct `env_name` as necessary.
+
+1. Confirm the following playbooks have already been run on target host before proceeding:
+  - users.yml
+  - dokku-server.yml
+  - ssmtp.yml
```

Comment:
```suggestion
Ensure the server has been set up according to standard practise documented in the root of this repository
```
Not sure of a reliable way to link to the repo root and this is sufficient.
By listing these things in every app, they are guaranteed to go out of date. So the goal is to have a canonical place where it should be documented, and refer to that each time.